flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -1,10 +1,15 @@
 class CDUAirportsMonitor {
-    static ShowPage(mcdu, reset = false) {
+    static ShowPage(fmc, mcdu, reset = false) {
+        mcdu.setCurrentPage(() => {
+            if (!this.frozen || !this.icao1) {
+                CDUAirportsMonitor.ShowPage(fmc, mcdu);
+            }
+        });
 
         // one delta t unit is about 1.5 ms it seems
         const update_ival_ms = 1000;
 
-        this.total_delta_t += mcdu._deltaTime;
+        this.total_delta_t += fmc._deltaTime; // TODO wut
```

Comment:
WUT! 🤔

Reply:
I will delete this code in the custom-fpm branch (where I am doing a general FMGC implementation of nearby airports that is shared amongst a number of different things).